db/migrate/20190706073144_create_page_unlock_promo_codes.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePageUnlockPromoCodes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :page_unlock_promo_codes do |t|
+      t.string :code
+      t.string :page_types
```

Comment:
Use the active record serializer to store an array instead of a pipe deprecated string.

Reply:
Deprecated = seperated....on my phone in a tent in Canada lol.